Minor fixes

withrocks opened the pull request:
- Disable failing tests, they will be enabled later
- Fix whitespace
- Add a check in the validate-setup script that checks if st2 is running as expected

Commit:
Don't run integration tests for bcl2fastq and siswrap

- These tests are currently failing and will be fixed later

Commit:
Fix whitespace

Commit:
Add st2ctl check in validate-setup

- This check should validate that all expected st2 services are up and running

johandahlberg commented:
Looks good. Maybe we should run: `validate-setup` as part of the ansible-playbook?

withrocks commented:
Sounds good.

Commit:
Add smoke test for services

- Check if the services are listening
- Exit the validation with an error message in red if failing
